### PR TITLE
docs: update draggable demo with rtl mode

### PR DESCRIPTION
**Pull request by miracles1919**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
可拖拽标签在 RTL mode 下样式问题
 
tabs-draggable

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: components/tabs/demo/custom-tab-bar-node.md [hunk 1]
```diff
@@ -152,4 +152,12 @@ ReactDOM.render(
   background: #fefefe;
   transition: all 0.3s;
 }
+
+.dragnode {
+  margin-right: 24px;
```
Comment (xrkffgg) [inline components/tabs/demo/custom-tab-bar-node.md:157]: margin-inline-end

Comment (afc163) [reply components/tabs/demo/custom-tab-bar-node.md:157]: master 不加 logical property。

Comment (miracles1919) [reply components/tabs/demo/custom-tab-bar-node.md]: done

**Comment by afc163** [comment]
Run `npm test -- components/tabs -u` in local to update snapshot.

**Comment by miracles1919** [comment]
done

**Pull request merged**